api/conf/application.conf

```diff
@@ -18,3 +18,6 @@ play.evolutions.db.default.useLocks = true
 
 ratings.tileserver.url = "http://localhost:8080"
 ratings.tileserver.url = ${?RATINGS_TILESERVER_URL}
+
+osrm.routing.uri = ${OSRM_ROUTING_URI}
+osrm.nearest.uri = ${OSRM_NEAREST_URI}
```

Comment:
Are these different enough to merit two separate lines? Why not have osrm.uri and then just use constants for the routing path vs the nearest path? or are the two very different?